x11-wm/compiz/Manifest

```diff
@@ -0,0 +1 @@
+DIST compiz-0.8.14.tar.xz 1137696 BLAKE2B 7a353bba661f5d81092f6850448524c44cdb2fbab1c65db994bf78dc193f131874c83664b64c461de9f4a63575af710088fdb2c0adcc3c6d0d4e06949c618ccc SHA512 ce02c7ac0ba864a68ace05abf4b9cb833185cf47a2dcaf5aa9e900336a4ecff5107ef503646bdd6ec675d9fa7a54bb5ae07df4f1cadea098765e5a5b4ca80d5e
```

Comment:
Don't call it update, it's a new package at this moment. Calling it 'update' makes me wth are older ebuilds.